internal/seclang/rule_parser.go

```diff
@@ -384,6 +373,60 @@ func ParseRule(options RuleOptions) (*corazawaf.Rule, error) {
 	return rp.rule, nil
 }
 
+func parseActionOperator(data string) (vars string, op string, actions string, err error) {
+	// So only need to TrimLeft below
+	data = strings.Trim(data, " ")
+	vars, rest, ok := strings.Cut(data, " ")
+	if !ok {
+		return "", "", "", fmt.Errorf("invalid format for rule with operator: %q", data)
+	}
+
+	rest = strings.TrimLeft(rest, " ")
+
+	if len(rest) == 0 || rest[0] != '"' {
+		return "", "", "", fmt.Errorf("invalid operator for rule with operator: %q", data)
+	}
+
+	op, rest, err = cutQuotedString(rest)
+	if err != nil {
+		return
+	}
+	op = utils.MaybeRemoveQuotes(op)
+
+	rest = strings.TrimLeft(rest, " ")
+	if len(rest) == 0 {
+		// No actions
+		return
+	}
+
+	if len(rest) < 2 || rest[0] != '"' || rest[len(rest)-1] != '"' {
+		return "", "", "", fmt.Errorf("invalid actions for rule with operator: %q", data)
```

Comment:
I wish this error would tell my why it is failing, something like `invalid actions for rule with unquoted operator: %q`?

Reply:
It could be missing one quote or both quotes and since we print the actual string as well, I think a more generic message is simpler here. If someone actually has a parse error here, likely there's a lot more wrong than just quotes